packages/dnb-eufemia/src/components/date-picker/hooks/useSubmittedCalendarDates.ts

```diff
@@ -0,0 +1,26 @@
+import { useCallback, useRef } from 'react'
+import { DatePickerDates } from './useDates'
+
+export type SubmittedCalendarDates = Pick<
+  DatePickerDates,
+  'startDate' | 'endDate'
+>
+
+export default function useSubmittedCalendarDates() {
+  const submittedCalendarDatesRef = useRef<SubmittedCalendarDates>({})
+
+  const setSubmittedCalendarDates = useCallback(
```

Comment:
```suggestion
  const setSubmittedDates = useCallback(
```

Do we need to have `Calendar` in the name? Does it give any extra value? 🤔 Just thinking out loud.

Reply:
I think not, based on the context. It's a mouthfull 😄